### PR TITLE
Fixes #175 - Sponsor UI Fixed

### LINKED ISSUE #175: Fixing the sponsor section UI
The sponsor logos which are of different sizes are currently not aligned properly and hence doesn't look good. A minor fix is needed to resolve this issue.

### PR DESCRIPTION
**Pull request by realslimshanky**
This fixes #175 

- The Sponsor Section UI has been fixed to better accommodate all the
sponsors
- A minor change in the size of the sponsor type heading is also
introduced

### CODE REVIEW

**Comment by realslimshanky** [comment]
@vipulgupta2048 I've fixed the issue with padding of sponsors image. I hope it looks good now.

**Comment by ananyo2012** [comment]
Is there any way to fix it without padding ? Using baseline alignment or vertical middle alignment. I know it's a bit tricky to implement but can you check once?

**Comment by realslimshanky** [comment]
@ananyo2012 the icons will still be a little off because of the sizes. Hence, I've carefully crafted the css in the PR.
image
image

**Comment by ananyo2012** [comment]
@realslimshanky Can you update the PR with the requested changes and solve the merge conflict ?

File: js/sponsor.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ constructSponsorBlock = function(data){
         var sponsorContent = "";
 
         $.each(data, function(key, value) {
-            sponsorContent += "<h3 class='text-center'>" + key + " Sponsors</h3>";
+            sponsorContent += "<br /><br /><h3 class='text-center' style='font-size:40px;'>" + key + "</h3>";
```
[the remark this answers is not in the document]
Comment (realslimshanky) [reply js/sponsor.js]: ACK

**Comment by realslimshanky** [comment]
@ananyo2012 all set. Ready for review.

**Comment by ananyo2012** [comment]
@realslimshanky An extra `undefined` class is added for sponsors that don't have a class

image

Can you make it conditional ?

File: custom/css/style.css [hunk 2]
```diff
@@ -1298,6 +1298,28 @@ ul.faq-answer li a:hover {
...
+    }
+
+    .pramati {
+        margin-top: 35px;
+    }
+
+    .livehealth {
+        margin-top: 35px;
+    }
+    .rechonsys {
```
Comment (ananyo2012) [inline custom/css/style.css]: Please fix the name `reckonsys` just to maintain consistency

**Comment by rajataaron** [comment]
@realslimshanky just do the sizing difference in sponsors like platinum sponsor logo should look bigger than gold sponsor and gold sponsor should look bigger than silver sponsor and so on.
cc: @ananyo2012 @vipulgupta2048

**Comment by ananyo2012** [comment]
@realslimshanky Can you take care of @rajataaron 's suggestion ?

File: custom/css/style.css (hunk 2 above)
Comment (realslimshanky) [reply custom/css/style.css]: ACK

**Comment by realslimshanky** [comment]
@rajataaron @ananyo2012 Please review. Below is how it looks now.
##### Platinum
image
##### Gold & Silver
image
##### Associate
image

**Pull request merged**